swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/Log.swift

```diff
@@ -185,6 +191,36 @@ private final class LogWriter {
     }
   }
 
+  private func ensureFileExists() -> Bool {
```

Comment:
Instead of returning a bool, why doesn't this just return the handle to write to? Then you can't forget to call this.

Reply:
updated